pc/storage/kdtree/kdtree.go

```diff
@@ -306,3 +363,19 @@ func (s *indiceSorter) Less(i, j int) bool {
 func (s *indiceSorter) Swap(i, j int) {
 	s.indice[i], s.indice[j] = s.indice[j], s.indice[i]
 }
+
+type neighborSorter struct {
+	neighbors []storage.Neighbor
+}
+
+func (ns *neighborSorter) Len() int {
+	return len(ns.neighbors)
+}
+
+func (ns *neighborSorter) Swap(i, j int) {
+	ns.neighbors[i], ns.neighbors[j] = ns.neighbors[j], ns.neighbors[i]
+}
+
+func (ns *neighborSorter) Less(i, j int) bool {
+	return ns.neighbors[i].DistSq < ns.neighbors[j].DistSq
+}
```

Comment:
```suggestion
type neighborSorter []storage.Neighbor

func (ns neighborSorter) Len() int {
	return len(ns)
}

func (ns neighborSorter) Swap(i, j int) {
	ns[i], ns[j] = ns[j], ns[i]
}

func (ns neighborSorter) Less(i, j int) bool {
	return ns[i].DistSq < ns[j].DistSq
}
```
and
```diff
-       sort.Sort(&neighborSorter{neighbors: neighbors})
+       sort.Sort(neighborSorter(neighbors))
```

Reply:
done in 053d68b2de03277c23c61fadb0a16367496dcc2a